common/source/docs/common-future-wiki-changes.rst

```diff
@@ -4,6 +4,21 @@
 Future Wiki Changes (in Master branch now)
 ==========================================
 
+:ref:`common-rpm` page
+----------------------
+
+- under types list at top: add ESC Telemetry
+
+- add section:
+
+ESC Telemetry - Average Motor RPM
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The RPM library can also be used to setup an 'RPM sensor' that computes and logs the average RPM for selected motors on the vehicle that are controlled by BLHeli_32 or BLHeli_S capable ESCs.  First the ESC telemetry will 
+need to be setup.  See :ref:`BLHeli Telemetry<common-dshot-blheli32-telemetry>` for details on how to do this.  Once complete set ``RPMx_TYPE`` to 4 and write the parameters to ArduPilot.  Then refresh/fetch the parameters.  You will find a number of additional parameters are now available for that instance.  Find and set ``RPMx_ESC_MASK`` to add which ESC channels you want to be included in the average. For example:
```

Comment:
Set RPMxTYPE to 5 for for ESC telemetry.  I supect that this is a hangover from when we went back and forth over removing/keeping type = 1 (PIN) in the type enum.  Have checked master and is definetly 5 now.

Reply:
done